autogen/agentchat/conversable_agent.py

```diff
@@ -1470,7 +1470,17 @@ def _generate_code_execution_reply_using_executor(
             # found code blocks, execute code.
             code_result = self._code_executor.execute_code_blocks(code_blocks)
             exitcode2str = "execution succeeded" if code_result.exit_code == 0 else "execution failed"
-            return True, f"exitcode: {code_result.exit_code} ({exitcode2str})\nCode output: {code_result.output}"
+
+            # create resulting string
```

Comment:
This will change the default behavior of agents. Can this be made optional through `code_execution_config`?

Reply:
Legacy `generate_code_execution_reply` had this function with `logs`. So I think it should be default behavior. To be honest I think it may be possibly be over-engineering to add it to `code_execution_config`. There aren't other configurations for controlling logging visibility in the project that I've found. I don't know if it would be all that intuitive of a property. What do you think about keeping it simpler? If you'd still want it, I'd make the prop `hide_code_file_output`, which feels out of place alongside `use_docker`, `work_dir`, `timeout` and `last_n_messages`. 
Also think it's unlikely it'd break any app regexing the output. If that's a concern I think there could also be an conversation about a better data structure than string to support api consistency (such as a metadata output property alongside `content`)